Classes/EventStore/Storage/Doctrine/DoctrineEventStorage.php

```diff
@@ -156,7 +156,7 @@ public function commit(StreamName $streamName, WritableEvents $events, int $expe
             }
 
             $this->connection->commit();
-        } catch (DBALException $exception) {
+        } catch (\Exception $exception) {
```

Comment:
More correctly, this should even be a `\Throwable`... but then that should maybe be changed at the other places too, to be in line.
```suggestion
        } catch (\Throwable $exception) {
```

Reply:
Good point, I'll solve that in a separate PR (see #242)